dotnet/src/Extensions/PromptTemplates.Prompty/PromptyKernelFunction.cs

```diff
@@ -0,0 +1,125 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.SemanticKernel.ChatCompletion;
+using Microsoft.SemanticKernel.Experimental.Prompty.Core;
+
+namespace Microsoft.SemanticKernel.Experimental.Prompty;
+public class PromptyKernelFunction : KernelFunction
```

Comment:
This doesn't need to be public, does it?

Reply:
nope, and the `PromptyKernelFunction` is removed